Source/FolioReaderPage.swift

```diff
@@ -425,7 +425,11 @@ public class FolioReaderPage: UICollectionViewCell, UIWebViewDelegate, UIGesture
     // MARK: UIMenu visibility
     
     override public func canPerformAction(action: Selector, withSender sender: AnyObject?) -> Bool {
-        if UIMenuController.sharedMenuController().menuItems?.count == 0 {
+		guard readerConfig.useReaderMenuController else {
+			return false
```

Comment:
Don't remove `guard` here, just return the super, so it will use the default menu.

```swift
guard readerConfig.useReaderMenuController else {
    return super.canPerformAction(action, withSender: sender)
}
```